client/state/signup/dependency-store/schema.js

```diff
@@ -0,0 +1,3 @@
+export const dependencyStoreSchema = {
+	type: 'object'
```

Comment:
Thanks for the schema overview documentation! I updated the schema to be more specific.